sass/components/Dashboard/utilization.scss

```diff
@@ -26,12 +26,20 @@
   text-align: right;
 }
 
-.kubevirt-utilization__item-narrow-chart {
+/* stylelint-disable plugin/selector-bem-pattern */
+.kubevirt-utilization__item-chart svg {
+  overflow: visible;
+}
+/* stylelint-enable */
+
+.kubevirt-utilization__item-chart--narrow {
   text-align: center;
 }
 
-.kubevirt-utilization__item-chart {
+.kubevirt-utilization__item-chart--wide {
   border-left-width: 1px;
   border-left-style: solid;
   border-left-color: rgb(209, 209, 209);
+  padding-right: 20px;
```

Comment:
rem/em?

Reply:
I actually removed the padding in latest patch as it seems to be not necessary